concordium-sdk/src/main/java/com/concordium/sdk/requests/getinstanceinfo/GetInstanceInfoRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.concordium.sdk.requests.getinstanceinfo;
```

Comment:
I don't think this type brings much value. Instead simply pass in the block hash and the contract address directly to the `getInstanceInfo` function.

Reply:
done